app/routes.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import { Switch, Route, Redirect } from 'react-router-dom';
 
-import Auth from './modules/Auth';
-import User from './modules/User';
-import AuthPage from './components/AuthPage';
-import AdminRoutes from '../admin/routes';
+import Auth from './auth/modules/Auth';
+import User from './auth/modules/User';
+import AuthPage from './auth/components/AuthPage/index';
```

Comment:
I think `import  AuthPage from './auth/components/AuthPage` will work. `index` is not needed.

Reply:
Oh this was done by Webstorm when I moved the file. I did not notice.